tests/loaders/css.test.js

```diff
@@ -0,0 +1,38 @@
+const {css} = require('../../loaders');
+
+const getConf = x => css(x).pop();
```

Comment:
Maybe we could add a small comment for clarity. Loaders could return multiple items in an array and if we also do that for the `css` loader, we need to remember to change this function

Reply:
We can actually return an object in the `css` loader instead of an array, so we wouldn't have to use `pop` anymore, which would make this function obsolete